.idea/jpa-buddy.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
why did you push this?

Reply:
Because I wanted both of us to have access to the same database